include/gatekeeper_lls.h

```diff
@@ -231,6 +237,10 @@ struct lls_config {
 	uint16_t          tx_queue_front;
 	uint16_t          rx_queue_back;
 	uint16_t          tx_queue_back;
+
+	/* Data structures used to limit the rate of icmp messages. */
+	struct token_bucket_ratelimit_state front_icmp_rs;
+	struct token_bucket_ratelimit_state back_icmp_rs;
```

Comment:
These fields are better off in `struct gatekeeper_if`.

Reply:
In the GK blocks, we also need to ratelimit the ICMP messages. However, each GK block maintains a data structure for rate limiting. Should we also maintain these two at the LLS block here?

Reply:
I just realized that you are following the GK model. Let's keep these fields here to keep LLS and GK in a similar fashion.